app/admin/applications/new/page.tsx

```diff
@@ -68,51 +53,30 @@ export default function NewApplicationPage() {
           />
         </div>
         <div>
-          <label htmlFor="appType" className="block mb-1">
-            Application Type
+          <label htmlFor="clientId" className="block mb-1">
+            Client ID
           </label>
-          <select
-            id="appType"
-            name="appType"
-            required
-            value={selectedAppType}
-            onChange={(e) => setSelectedAppType(e.target.value as AppType | "")}
+          <input
+            type="text"
+            id="clientId"
+            name="clientId"
             className="w-full p-2 border rounded text-black"
-          >
-            <option value="">Select an application type...</option>
-            {AppTypes.map((type) => (
-              <option key={type} value={type} className="capitalize">
-                {type.toLowerCase().replace(/_/g, " ")}
-              </option>
-            ))}
-          </select>
+          />
+        </div>
+        <div>
+          <label htmlFor="clientSecret" className="block mb-1">
+            Client Secret
+          </label>
+          <input
+            type="password"
+            id="clientSecret"
+            name="clientSecret"
+            className="w-full p-2 border rounded text-black"
+          />
         </div>
-
-        {(() => {
-          switch (selectedAppType) {
```

Comment:
Wait we still want appType in the frontend and api layer, just not in the database